contracts/0.8.9/WithdrawalQueueERC721.sol

```diff
@@ -16,6 +16,18 @@ import {Strings} from "@openzeppelin/contracts-v4.4/utils/Strings.sol";
 import {IWstETH, WithdrawalQueue} from "./WithdrawalQueue.sol";
 import {AccessControlEnumerable} from "./utils/access/AccessControlEnumerable.sol";
 import {UnstructuredRefStorage} from "./lib/UnstructuredRefStorage.sol";
+import {UnstructuredStorage} from "./lib/UnstructuredStorage.sol";
+
+/**
+  * @title Interface defining INFTDescriptor to generate ERC721 metadata
```

Comment:
Nitpicking alert. Actually it provides only `tokenURI`, because ERC721 metadata includes name and symbol as well.